### PR TITLE
Add some nested diagrams

### PR DESCRIPTION
**Pull request by kfox1111**

### CODE REVIEW

File: examples/nested/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Possible Nesting Configurations
+
+There are multiple ways of configuring the chart depending on what you want to use nesting for.
+
+## Diagram for nesting across Kubernetes clusters:
```
Comment (faisal-memon) [inline examples/nested/README.md:5]:
```suggestion
## Nesting across Kubernetes clusters:
```

File: examples/nested/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+![Multiple Kubernetes Cluster](./multicluster.png)
+
+## Diagram using nesting within a Kubernetes cluster:
```
Comment (faisal-memon) [inline examples/nested/README.md:8]:
```suggestion
## Nesting within a Kubernetes cluster:
```

File: examples/nested/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Possible Nesting Configurations
```
Comment (faisal-memon) [inline examples/nested/README.md:1]: Can you add a small blurb about using `namesapceOverride` to get the components into different namespaces?

A small pros/cons discussion of when to use the different approaches would be nice too.

Comment (kfox1111) [reply examples/nested/README.md:1]: Was trying to take an iterative approach here. Getting solid examples of these configurations will take quite a bit of time to get right and document. But the diagrams are useful to show off what is currently possible with the chart. So, maybe 2  pr's?

Comment (faisal-memon) [reply examples/nested/README.md:1]: Yea thats fine.

**Pull request merged**